ebpf/go.mod

```diff
@@ -25,6 +25,7 @@ require (
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bufbuild/connect-go v1.10.0 // indirect
```

Comment:
?

Reply:
It is complicated, but it is because of that line: https://github.com/grafana/pyroscope/blob/main/ebpf/go.mod#L12

I think I need to merge this first, then tag a new api version and then I can update the ebpf packgae